graphix/gflow.py

```diff
@@ -848,3 +1482,21 @@ def get_output_from_flow(flow: dict[int, set]) -> set:
         non_inputs |= correction
     outputs = non_inputs - non_outputs
     return outputs
+
+
+def get_pauli_nodes(meas_planes: dict[int, str], meas_angles: dict[int, float]) -> tuple[set[int], set[int], set[int]]:
+    Lx, Ly, Lz = set(), set(), set()
+    for node, plane in meas_planes.items():
+        if plane == "XY" and meas_angles[node] == int(meas_angles[node]):  # measurement angle is integer
+            Lx |= {node}
+        elif plane == "XY" and 2 * meas_angles[node] == int(2 * meas_angles[node]):  # measurement angle is half integer
+            Ly |= {node}
+        elif plane == "ZX" and meas_angles[node] == int(meas_angles[node]):
+            Lz |= {node}
+        elif plane == "ZX" and 2 * meas_angles[node] == int(2 * meas_angles[node]):
+            Lx |= {node}
+        elif plane == "YZ" and meas_angles[node] == int(meas_angles[node]):
+            Ly |= {node}
+        elif plane == "YZ" and 2 * meas_angles[node] == int(2 * meas_angles[node]):
+            Lz |= {node}
+    return Lx, Ly, Lz
```

Comment:
The notation of the planes seem slightly different. We are now following Backens paper[2021]. Please see the [Backens et al(2021)](https://quantum-journal.org/papers/q-2021-03-25-421/) p.8 or the `Plane` object in `pauli.py`.